README.md

```diff
@@ -110,188 +133,30 @@ Salary slip kata
            Taxable income: £12,500.00
            Tax Payable: £4,466.67
 
-#### Scenario 9: ...for an employee earning an annual salary of £160,000.00   
-  ...
-  <p>Then the monthly salary slip should contain the below:</p>
-  
+Calculation rules:
+ * Tax-free allowance: When the Gross Salary exceeds £100,000.00, the tax-free allowance starts decreasing. It decreases by £1 for every £2 earned over £100,000.00. And this adjusted excess is taxed at the Higher rate tax.
```

Comment:
I'll remove the word `adjusted` from master, so please remove it from here as well